src/EditorFeatures/Core/Implementation/Workspaces/ProjectCacheService.cs

```diff
@@ -17,19 +17,15 @@ internal partial class ProjectCacheService : IProjectCacheHostService
         private readonly Workspace _workspace;
         private readonly Dictionary<ProjectId, Cache> _activeCaches = new Dictionary<ProjectId, Cache>();
 
-        private readonly SimpleMRUCache _implicitCache = new SimpleMRUCache();
+        private readonly SimpleMRUCache _implicitCache;
         private readonly ImplicitCacheMonitor _implicitCacheMonitor;
 
-        public ProjectCacheService(Workspace workspace, int implicitCacheTimeout, bool forceCleanup = false)
+        public ProjectCacheService(Workspace workspace, int implicitCacheTimeout)
         {
             _workspace = workspace;
 
-            // forceCleanup is for testing
-            if (workspace?.Kind == WorkspaceKind.Host || forceCleanup)
-            {
-                // monitor implicit cache for host
-                _implicitCacheMonitor = new ImplicitCacheMonitor(this, implicitCacheTimeout);
-            }
+            _implicitCache = new SimpleMRUCache();
+            _implicitCacheMonitor = new ImplicitCacheMonitor(this, implicitCacheTimeout);
```

Comment:
Now that the ```if``` has been removed, we have a test that covers all code paths (```TestImplicitCacheMonitoring```).  If we add the ```if``` back, then yeah, we should have a test for other types of Workspaces.